Detectors/FIT/FT0/base/src/Geometry.cxx

```diff
@@ -1,122 +1,126 @@
-                          -9.026849243816981, 15.038129472387304, -15.038129472387304,
-                          15.003621961057961, -15.003621961057961};
-  Double_t yi[NCellsC] = {3.1599494336464455, -3.1599494336464455,
-                          9.165191680982874, -9.165191680982874, 3.1383331772537426,
-                          -3.1383331772537426, 9.165226363918643, -9.165226363918643,
-                          15.141616002932361, -15.141616002932361, 9.16517861649866,
-                          -9.16517861649866, 15.188854859073416, -15.188854859073416,
-                          9.165053319552113, -9.165053319552113, 15.188703787345304,
-                          -15.188703787345304, 3.138263189805292, -3.138263189805292,
-                          9.165104089644917, -9.165104089644917, 15.141494417823818,
-                          -15.141494417823818, 3.1599158563428644, -3.1599158563428644,
-                          9.165116302773846, -9.165116302773846};
-
-  Double_t zi[NCellsC];
-  for (Int_t i = 0; i < NCellsC; i++) {
-    zi[i] = TMath::Sqrt(TMath::Power(crad, 2) - TMath::Power(xi[i], 2) - TMath::Power(yi[i], 2));
-  }
-
-  // get rotation data
-  Double_t ac[NCellsC], bc[NCellsC], gc[NCellsC];
-  for (Int_t i = 0; i < NCellsC; i++) {
-    ac[i] = TMath::ATan(yi[i] / xi[i]) - TMath::Pi() / 2 + 2 * TMath::Pi();
-    if (xi[i] < 0) {
-      bc[i] = TMath::ACos(zi[i] / crad);
-    } else {
-      bc[i] = -1 * TMath::ACos(zi[i] / crad);
-    }
-  }
-  Double_t xc2[NCellsC], yc2[NCellsC], zc2[NCellsC];
-
-  // compensation based on node position within individual detector geometries
-  // determine compensated radius
-  Double_t rcomp = crad + mStartC[2] / 2.0; //
-  for (Int_t i = 0; i < NCellsC; i++) {
-    // Get compensated translation data
-    xc2[i] = rcomp * TMath::Cos(ac[i] + TMath::Pi() / 2) * TMath::Sin(-1 * bc[i]);
-    yc2[i] = rcomp * TMath::Sin(ac[i] + TMath::Pi() / 2) * TMath::Sin(-1 * bc[i]);
-    zc2[i] = rcomp * TMath::Cos(bc[i]);
-
-    // Convert angles to degrees
-    ac[i] *= 180 / TMath::Pi();
-    bc[i] *= 180 / TMath::Pi();
-    gc[i] = -1 * ac[i];
-    mAngles[i].SetXYZ(ac[i], bc[i], gc[i]);
-    mMCP[i + NCellsA].SetXYZ(xc2[i], yc2[i], zc2[i]);
-  }
-}
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include "FT0Base/Geometry.h"
+#include "TSystem.h"
+#include "Framework/Logger.h"
+#include <sstream>
+#include <iomanip>
+#include <string>
+#include <iostream>

```

Comment:
I don't understand why these lines appear as "changed".

Reply:
You have probably introduced additional white space characters. Would you be able to fix this and provide a cleaned up version?

Reply:
I am not sure why it triggered these lines as "changed".  I looked into the file and they do not appear to have additional white space.